content/api/vision/_index.md

```diff
@@ -20,6 +20,7 @@ The markers in the list have some useful attributes:
 - `pixel_centre` - returns the location in pixels of the centre of the marker in the captured image.
 - `cartesian` - returns details of the position of the marker in the [Cartesian](coordinates/#cartesian-coordinates) coordinate system.
 - `spherical` - returns details of the position of the marker in a [spherical](coordinates/#spherical-coordinates) coordinate system.
+- `orientation` - returns the rotation of the marker around its centre. See [Orientation](coordinates/#orientation) for its coordinate system.
```

Comment:
I don't really like the additional sentence just to link, could we word orientation into the initial one. Perhaps:
> returns the [Orientation] of the marker around its centre.

Reply:
fixed in 55660070206d6112a261eaaecf3e7a20fbfeb12b